common/static/sass/edx-pattern-library-shims/_buttons.scss

```diff
@@ -0,0 +1,121 @@
+// ------------------------------
```

Comment:
@bradenmacdonald Well, it will be useful when it lands; until than we can't use it. And after it lands it's just removing 5-6 styles from DnDv2 own css anyway.

Oh, and I don't think we want to convert DnDv2 css to sass/scss, at least as part of  edx-solutions/xblock-drag-and-drop-v2#86